containers/python-3/.devcontainer/devcontainer.json

```diff
@@ -1,15 +1,17 @@
 {
 	"name": "Python 3",
 	"context": "..",
-	"dockerFile": "Dockerfile",
+
+	// Update the FROM statement in this Dockerfile to pick a version of Python.
+	"dockerFile": "base.Dockerfile",
 
 	// Set *default* container specific settings.json values on container create.
 	"settings": { 
 		"terminal.integrated.shell.linux": "/bin/bash",
 		"python.pythonPath": "/usr/local/bin/python",
 		"python.linting.enabled": true,
 		"python.linting.pylintEnabled": true,
-		"python.linting.pylintPath": "/usr/local/bin/pylint"
+		"python.linting.pylintPath": "/usr/local/py-utils/bin/pylint"
```

Comment:
If we are going to install all the tools, would it be worth setting the paths now? They will be inactive unless the user flips them on, but at least they wouldn't need to specify the paths themselves (although even specifying them isn't necessary if they can be found on `PATH`).

Reply:
Can do.